e-learning-services/src/main/resources/db/changelog/elearning-rdbms.db.changelog-1.0.0.xml

```diff
@@ -121,5 +121,17 @@
 
 
 	</changeSet>
+	
+	<changeSet author="elearning" id="1.0.0-5">
+
+
+		<addColumn tableName="ADDON_E_LEARNING_THEME">

```

Comment:
You can add the column explicitly in the create table changelog, since it is not in production

Reply:
In my case, if I put directly in the create table changelog, I'd have to delete the database and create it again for the changes to apply, and since I didn't want to lose the data I already created, I kept it that way.

Reply:
Yes in your case you can add the column as it suits you but when it comes to the commit you should add the column as i mentioned